### ISSUE TITLE
NTR: observing system

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/COB#190]
ENVO has a class **observing system** http://purl.obolibrary.org/obo/ENVO_01001469 
_A system of constructed and manufactured products which are used by humans to produce data, information, or knowledge about material, immaterial, or processual entities._
cc @pbuttigieg

This has a single child (observatory) and some grandchildren

![image](https://user-images.githubusercontent.com/50745/145657995-17b97cf1-6ab2-474f-b945-0bf1fe34c5b9.png)

This is a bit problematic - the definition of observing system is broad enough that it should encompass many classes in OBI (@bpeters42), and machines such as MRI machines, yet users may be confused not to see OBI classes under here

The parent term 'system' is not in COB (I am responsible for the class 'system' in RO, which was introduced by Barry in https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0097-6 -- Barry said this term would make it's way into BFO)

I think there is room for a very general class in COB that can cover a variety of material entities that have a role in sensing for investigative purposes. I would strongly recommend this have a logical definition with a simple pattern, such that different ontologies that have such sensory-investigative systems will have terms auto-classified here.

Alternatively OBI could adopt this entire branch of ENVO (via obsoletion-replacement), but a challenge here is there are users e.g. NASA who want this for Planetary Science, Heliophysics, Biology, Physical Science and Earth Science... (@rduerr)

**Comment by bpeters42** [comment]
As said elsewhere, we should create a COB:device class that serves as a broad parent, and 'detection device' or something like that as a child that is defined by its function and should serve all of those purposes. We had talked about this in the OBI calls, and as I recall (but I DONT speak for obi) there were no objections. 

I do want to point out that device / instrument / detectors are defining classes of what OBI was created for, and we have decades of work on these terms. So this would not be something done lightly 

- Bjoern

**Comment by ddooley** [comment]
So an observing system is a set of material things used by an observing process to output observations?  Connected to this, in another draft paper I was proposing that we have:

_`An observation: An observation is a data item resulting from an invasive or non-invasive observation process occurring at an instant or duration of time, and includes at least one measure that (semantically) “is about” a characteristic of an entity (for example Jane Doe’s weight). Other aspects of an observation may be documented: the particular time and context it happened in, and the sensor (device or human or other organism organ) that was involved.`_ 

This is accompanied by an analysis of SOSA entities and relationships and what OBO was missing (e.g. "sensor") in that mix.  So this move to add observation system is very opportune. In diagram below, "assay" could be replaced with a more general "observation process"?
![image](https://user-images.githubusercontent.com/4000582/145745913-721ba9c8-13ba-4ae9-b516-9108da0912ac.png)

**Comment by ddooley** [comment]
Ok!  But to be clear I meant create a new term "observation process" and put it into the diagram.  But only if the new term is clearly broader in scope (and therefore parent of) "assay".

**Comment by DanBerrios** [comment]
@ddooley  I agree...  Seems correct to add observation process as parent of assay, and as a child of planned process, and also import ENVO observing system and make device or investigative-device a parent of observing system.  No?

GeneLab uses "assay", and it should stay I think...not every observing process is going to be called an assay.

**Comment by bpeters42** [comment]
I do want to make sure that I don't come across as trying to shut down this discussion; but I do want to point out where we have discussed the very same questions before. The problem has been defining what distinguishes an 'assay' from an 'observing process'. An example for the former being testing a blood sample for its glucose concentration. An example for the later being be counting birds in a habitat. It turns out to be very hard to define the boundary. So we ended up calling the general process 'assay'. If there is a general 'observing process' it needs to be clear how it is different from the current definition of 'assay' which is: 'A planned process with the objective to produce information about the material entity that is the evaluant, by physically examining it or its proxies'.

**Comment by ddooley** [comment]
I hear you, and indeed assay may be satisfactory because of its broadly scoped definition.  

One thing I'm wondering in a devil's advocate kind of way about material entity being hard-wired into observation process/assay:  What if I or an algorithm are looking for (observing) circles in an image.  Do I have to change the vocabulary I'm using if I learn that the image (which is usually digital these days anyways) was simulated?  Or can I keep my observation process language and its outputs, and just say the input was simulated?

**Comment by bpeters42** [comment]
Good points @ddooley . The purely data-driven process could be considered a data-transformation (e.g. image recognition). And if the initial image is generated from a real thing, the combined image generating and image recognition process could be an assay (e.g. detection of lesions in the lung using image detection on an X-Ray picture). But the image recognition part can also be run on simulated data. These kind of questions of where a assay starts and stops and if data transformations are part of it are one of the topics for the upcoming OBI workshop, and are part of @zhengj2007  session.

**Comment by wdduncan** [comment]
There is a lot of interesting discussion here about observation processes :)  

@ddooley some comments about your proposed definition for [observation](https://github.com/OBOFoundry/COB/issues/190#issuecomment-992065633):  
- It states "resulting from an invasive or non-invasive observation process". Would it be simpler to just say it results from an observation process? In other words, are there other observation processes that are not either invasive or non-invasive?
- I have a minor concern (from a language perspective) about observations/data items getting confused with the things that they are observations of. For example, I have a particular height, a quality that inheres in me. An observation process can (of course) produce data about my height, but my height (in the sense of quality) is not a data item.
- In the height example, I would have modeled the data item about my height as a `measurement datum` (assuming my height was measured with a device). Under your proposal, I think this would entail making `measurement datum` a subtype of observation. Is this something you want?
- The observation "is about" the characteristic of the entity, but should it also be about the entity itself? Is an observation about my height also about me?
- In the diagram, there is an `observable property` concept. Doesn't this cover all qualities? In other words, is there a distinction between `observable property` and `quality`? I think you can argue that `realizables` aren't directly observable (only their manifestations are), but such a position may lead into a quagmire of arguing about direct vs indirect observations :(

**Comment by ddooley** [comment]
That paragraph, out of a paper being reviewed which is aimed at non BFO experts interested in food processing modelling, is defining a data structure semantic used in the paper, but it has needs for an ontology to meet too.  Here's the context of some related terms, offered up in the spirit of "trial by fire!" (e.g. I'm wondering how not to be circular about defining characteristic without going BFO!).  

> - A characteristic: a “characteristic”, “feature”, “quality”, “attribute” or “phenotype” are often used to describe an observable property of an object. In food science “quality” may be used as a value judgement of a product, like a “good quality” ripe peach.  Within OBO, BFO uses “quality” rather than characteristic for an observable object property, and so we also frequently use that term and sense below. 

> - A measure (aka measurement): a data item record of a categorical, numeric, or numeric and unit value. The unit may also reveal a dimension of the thing being measured, for example if unit is “gram” then a material entity is being measured for mass. Measures may be simulated or predicted, or subject to precision and accuracy, and may be the result of faulty or miscalibrated equipment. This paper takes the position that a measure does not include time, place, or sensor “aboutness” information. 

> - An observation: An observation is a data item resulting from an invasive or ...

> An observation data item requires a data structure to encompass the contextual information about the time and place etc. of a measure. A data set composed of these observations could adopt the characteristic values they have in common, for example, if all of a dataset’s observations were made at the same location, then that location can become a characteristic of what the dataset is about, and is inherited rather than repeated at the observation level. 

Hopefully echoing natural language semantics, observation is being explored as a multi-component data structure which is able to have one or more measures as parts, a number of which can be about the context of the observation rather than just characteristics of the thing being measured. 

- I agree your "results from an observation process" is fine; one could then have two subclasses of destructive or non-destructive assays/observation processes if desired.
- I agree about observation process vs outputted data items.
- About 'measurement datum' being a subtype of observation.  As shown above I'd like to have something like a "measure" which isn't necessarily about anything else or its qualities.  I'd like to be able to talk about a set of measures, and their average say, without necessarily wiring in that they be about material entities or simulated ones.  Then we introduce an  observation which does involve at least one measure that is necessarily about something else.  Ok?  I realize that rocks the OBI boat, something maybe to raise in upcoming workshop.
- I totally like the idea that aboutness is somewhat transitive across processes and various relations.  Indeed if a datum is about your height, then it is about you.  Its also about the measuring device and process that outputted the datum.  Its about what that measuring device measured.  But has anyone played with formalizing inference axioms about that? I haven't.
- That "observable property" and everything in above diagram to left of dashed line is SOSA's ontology, and I'd say yes it seems to cover all qualities. I'm saying all the OBO stuff to the right is the stand-alone equivalent.

This diagram visualizes some of these constructs at play (p.s. I don't claim that it is all OBO in paper):
![image](https://user-images.githubusercontent.com/4000582/149878953-79106482-7a6b-4340-a6a1-2c6d177fddc8.png)
